protocol/avdl/keybase1/tlf_keys.avdl

```diff
@@ -4,6 +4,12 @@
 // getting TLF keys
 protocol tlfKeys {
   import idl "common.avdl";
+  import idl "identify.avdl";
+
+  enum TLFIdentifyBehavior {
+    DEFAULT_KBFS_1,
```

Comment:
i would set the default to be 0, that way older clients will do the right thing by default.